census/google_calendar.py

```diff
@@ -28,9 +28,9 @@ def google_publish_event(event):
         'dateTime': event.end_datetime.isoformat(),
         'timeZone': 'America/Los_Angeles',
       },
-      #'recurrence': [
-      #  'RRULE:FREQ=DAILY;COUNT=2'
-      #],
+      'recurrence': None if str(event.recurrences) == '' else [
```

Comment:
In the case where `str(event.recurrences) == ''`, is `event.recurrences == None`? That would simplify the ternary.

I remember in the UI you could add multiple recurrence rules.. Do we disable that, or is there a case where we would be passing multiple recurrences in this payload?

Reply:
== None does not see to trigger TRUE.
I don't know how we would disable multiple rules.
I have tested with multiple rules and it does work (one test only)